app/models/ontology.rb

```diff
@@ -5,46 +5,29 @@ class Ontology < ActiveRecord::Base
   include Metadatable
 
   # Ontology Model Includes
+  include GraphStructures::SpecificFetchers::Mappings
+  include IRIUrlBuilder::Includeable
+  include Ontology::Associations
```

Comment:
I would've thought that associations and attributes belong together, seeing as they together form the structure of the model.